### PR TITLE
IAM-458-Bump microk8s version in CI

### PR DESCRIPTION
**Pull request by bencekov**
Bumped microk8s version to 1.28-strict/stable in the CI integration test.

### CODE REVIEW

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
         uses: charmed-kubernetes/actions-operator@main
         with:
           provider: microk8s
-          channel: 1.25-strict/stable
+          channel: 1.28-strict/stable
```
Comment (shipperizer) [inline .github/workflows/tests.yaml:33]:
```suggestion
          channel: 1.{{ matrix.version }}-strict/stable
```

just wondering if @natalian98 wanted to only bump or test on different releases

If that's the case adding also the below on the job config will make it happen

```
strategy:
      matrix:
        version: [25,26,27,28]
```

Comment (natalian98) [reply .github/workflows/tests.yaml:33]: Maybe we could test individual charms just on 1.28 and the bundle on 1.27 and 1.28? I'm worried that if we test on 4 versions, the CI is going to take too long.

Comment (bencekov) [reply .github/workflows/tests.yaml:33]: I agree, I think the integration test take too long already.

**Review by natalian98 — APPROVED** [review]

File: .github/workflows/tests.yaml (hunk 1 above)
Comment (shipperizer) [reply .github/workflows/tests.yaml:33]: they would run in parallel, but the bundle tests thing sounds a good compromise

**Pull request merged**